Tests/gui/GSCodingFlags/GSCellFlags.m

```diff
@@ -0,0 +1,62 @@
+#include "Testing.h"
+
+#include <Foundation/NSAutoreleasePool.h>
+#include <Foundation/NSValue.h>
+
+#include <AppKit/NSApplication.h>
+#include "GSCodingFlags.h"
+
+int main()
+{
+    CREATE_AUTORELEASE_POOL(arp);
+    GSCellFlagsUnion mask = { { 0 } };
+
+    START_SET("GSCodingFlags GNUstep CellFlags Union")
+    
+    NS_DURING
+    {
+      [NSApplication sharedApplication];
+    }
+    NS_HANDLER
+    {
+      if ([[localException name] isEqualToString: NSInternalInconsistencyException ])
+        SKIP("It looks like GNUstep backend is not yet installed")
+    }
+    NS_ENDHANDLER
+    // first make sure flags translate to values
+    mask.flags.state = 1;
+    mask.flags.selectable = 1;
+    mask.flags.scrollable = 1;
+    mask.flags.editable = 1;
+    mask.flags.continuous = 1;
+    mask.flags.useUserKeyEquivalent = 1;
+    mask.flags.truncateLastLine = 1;
+
+#if GS_WORDS_BIGENDIAN == 1
+    pass(mask.value == 0b00010010000000000001110000001001, "mask.flags translates to mask.value");
+#else
+    pass(mask.value == 0b10010000001110000000000001001000, "mask.flags translates to mask.value");
+#endif
+    // reset mask
+    mask.value = 0;
+    mask.flags = (GSCellFlags){0};
+    // now make sure values translate to flags
+#if GS_WORDS_BIGENDIAN == 1
+    mask.value = 0b00000001001000000000110100000001;
```

Comment:
Both values look wrong. I would expect that they are different and identical to the ones above